upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -273,6 +289,12 @@ func (_ *Subnet) RenderTerraform(t *terraform.TerraformTarget, a, e, changes *Su
 		return t.AddOutputVariableArray("subnet_ids", terraformWriter.LiteralFromStringValue(*e.ID))
 	}
 
+	if strings.HasPrefix(aws.StringValue(e.IPv6CIDR), "cidrsubnet-") {
+		// TODO: Implement using "cidrsubnet"
+		// https://www.terraform.io/docs/language/functions/cidrsubnet.html
+		return fmt.Errorf("<cidrsubnet> in not supported with Terraform target: %q", aws.StringValue(e.IPv6CIDR))
```

Comment:
Let me know if you need help adding support for `cidrsubnet()`.

Looking at the literal, writer, and hcl2 code right now, it is a bit more involved than I would like. I think we could refactor the `file()`/`filebase64()` functionality to more easily support arbitrary functions like `cidrsubnet()`